demo/test.c

```diff
@@ -902,6 +923,10 @@ static int test_mp_prime_is_prime(void)
       if (cnt == MP_NO) {
          printf("\nsub is not prime!\n");
       }
+      mp_prime_frobenius_underwood(&b, &fu);
+      if (fu == MP_NO) {
+         printf("\nfrobenius-underwood says sub is not prime!\n");
```

Comment:
Uuh or are the different tests inconsistent?

Reply:
nope, not that I've noticed, it's just that `mp_prime_frobenius_underwood()` was never called in the default configuration